src/midi.ts

```diff
@@ -15,9 +15,10 @@ import * as jsmidgen from 'jsmidgen';
 export const midi = (
   notes: NoteObject[],
   fileName: string | null = 'music.mid',
-  bpm?: number
+  bpm?: number,
+  trackName?: string
```

Comment:
When more than 3 args are required, it s usually time to refactor. For now we can have it this way, but in v4 we ll make this a object literal input to the midi method.

Reply:
The problem with using fileName is that it contains the full path of the file, which means the MIDI clip title will look something like: "./path/to/file". I did consider using a regex to filter out the fileName from the path but figured it might be nicer to provide an option to name the file and the MIDI clip differently if so desired. It's totally your call if you want to add that functionality though, happy to change it!

Reply:
Yea better to use the filename.. you can strip off the path and extension with regex.

Reply:
You can do something on the lines of https://jsbin.com/zociyayemu/edit?js,console